### PR TITLE
Update Twitter cards

### LINKED ISSUE #63: Show cover image as social preview for posts
Currently, only a small generic preview is shown.

Screenshot 2022-02-01 at 21 22 22

We should use the `summary_large_image` tag for twitter cards to get the large preview.

### PR DESCRIPTION
**Pull request by dnlggr**
Resolves #63 and #64.

## 📸 

Screenshot 2022-02-02 at 15 56 04

### CODE REVIEW

File: src/pages/podcast/[shortcode]/[[...slug]].js [hunk 1]
```diff
@@ -202,7 +202,7 @@ export default function Episode({ episode, isShortLink }) {
             <meta name="og:description" property="og:description" content={episode.description} />
             <meta name="og:image" property="og:image" content={episode.image} />
 
-            <meta name="twitter:card" content="summary" />
+            <meta name="twitter:card" content="summary_large_image" />
```
Comment (dnlggr) [inline src/pages/podcast/[shortcode]/[[...slug]].js:205]: Not sure if this is the right choice. See #64.

Comment (dergigi) [reply src/pages/podcast/[shortcode]/[[...slug]].js:205]: Yep, that should be correct!

Comment (dergigi) [inline src/pages/podcast/[shortcode]/[[...slug]].js:205]:
```suggestion
            <meta name="twitter:card" content="summary" />
```

**Review by dergigi — APPROVED** [review]
Just checked the card previews for the preview build for both the podcast episodes and the blog posts, and it looks good. Large for blog posts, regular for episodes. ✅

**Pull request merged**